src/pyfaf/actions/attach_centos_bugs.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2015  ABRT Team
+# Copyright (C) 2015  Red Hat, Inc.
+#
+# This file is part of faf.
+#
+# faf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# faf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with faf.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyfaf.actions import Action
+from pyfaf.bugtrackers import bugtrackers
+from pyfaf.queries import (get_mantis_bug,
+                           get_report_by_hash,
+                           get_reportmantis,
+                           get_osrelease)
+from pyfaf.storage import ReportMantis
+
+
+class AttachCentosBugs(Action):
+    """
+    Looks up all bugs created using ABRT on CentOS MantisBT and attaches them
+    to reports using the bthash from URL. This is a workaround until proper
+    attaching is supported by the ABRT client.
```

Comment:
> No I'm saying you've deployed an incomplete feature and you're doing this over and over again.

You are right. That's my fault.

The attaching was designed:
https://github.com/abrt/libreport/issues/304
and it had to be temporarily disabled:
https://github.com/abrt/libreport/commit/f111a471853c7077127eb9b07c9adaa14c5f178c 

I've made these decisions and I took full responsibility for them.

> I prefer automation and I've invested a LOT of time into it so I don't have to do things like test days..

I really appreciate your efforts. You have done a great work! I use it a lot and I add new test cases for bugs and features. But I also appreciate things like test days.

> Like what? Adding new features?

It might appear I like new features but I'd rather stabilize the project and fix the bugs. Unfortunately, I am an loyal employee and my employers often have different orders for me. I cannot recollect any new feature I've develop where I'm also author and initiator.

> Maybe use your head as well from time to time.

I do use my head but that's obviously not enough.  Hence, I'm kindly asking you to ACK or NACK all my future pull request.